onnxruntime/test/perftest/main.cc

```diff
@@ -43,6 +43,12 @@ int real_main(int argc, char* argv[]) {
   }
   std::random_device rd;
   perftest::PerformanceRunner perf_runner(env, test_config, rd);
+
+  // Exit if user enabled -n option so that they can just the optimized model or EP context model
```

Comment:
let's not comment this way. the official use case here is for measuring session creation time to measure impact of enabling any initialization optimizations.